.github/workflows/fuzzing.yml

```diff
@@ -0,0 +1,77 @@
+name: Fuzzing
+
+on:
+  # Regression testing
+  push:
+    branches: 
+      - v2
+  pull_request:
+    branches: 
+      - v2
+
+  # Daily midnight fuzzing
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  fuzzing:
+    name: Fuzzing
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Download go-fuzz tools and the Fuzzit CLI, move Fuzzit CLI to GOBIN
+      if: github.repository == 'caddyserver/caddy'
+      run: |
+        # Install Clang-7.0 because other versions seem to be missing the file libclang_rt.fuzzer-x86_64.a
+        sudo add-apt-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+        sudo apt update && sudo apt install -y clang-7 lldb-7 lld-7
+
+        go get -v github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+        wget -q -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.77/fuzzit_Linux_x86_64
+        chmod a+x fuzzit
+        mv fuzzit $(go env GOPATH)/bin
+
+    - name: Download go-fuzz tools and the Fuzzit CLI, move Fuzzit CLI to GOBIN
```

Comment:
The name needs to be updated to match the actual content

Reply:
Yep, was already doing that 😄 just forgot to fix that one when copy-pasting from the azure yml